<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlobDtection_Gui</class>
 <widget class="QWidget" name="BlobDtection_Gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="SaveConfig_Button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Config</string>
   </property>
  </widget>
  <widget class="QComboBox" name="BlobDetection_comboBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>SimpleBlobDetect</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GassianBlobDetect</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;BLOB DETECTION ALOGRITHM&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadConfig_Button">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>520</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Config</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>101</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTextEdit" name="minThreshold_textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="maxThreshold_textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="minArea_textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="maxArea_textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="minConvexity_textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="maxConvexity_textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="minCircularity_textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="maxCircularity_textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="minInertiaRatio_textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="maxInertiaRatio_textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>121</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="minThreshold_label">
      <property name="text">
       <string>minThreshold</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="maxThreshold_label_2">
      <property name="text">
       <string>maxThreshold</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="minArea_label">
      <property name="text">
       <string>minArea</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="maxArea_label">
      <property name="text">
       <string>maxArea</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="minConvexity_label">
      <property name="text">
       <string>minConvexity</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="maxConvexity_label">
      <property name="text">
       <string>maxConvexity</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="minCircularity_label">
      <property name="text">
       <string>minCircularity</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="maxCircularity_label">
      <property name="text">
       <string>maxCircularity</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="minInertiaRatio_label">
      <property name="text">
       <string>minInertiaRatio</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="maxInertiaRatio_label">
      <property name="text">
       <string>maxInertiaRatio</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>431</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSlider" name="minThreshold_horizontalSlider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="maxThreshold_horizontalSlider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="minArea_horizontalSlider">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="maxArea_horizontalSlider">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="minConvexity_horizontalSlider">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="maxConvexity_horizontalSlider">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="minCircularity_horizontalSlider">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="maxCircularity_horizontalSlider">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="minInertiaRatio_horizontalSlider">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="maxInertiaRatio_horizontalSlider">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
